public/index.html

```diff
@@ -4,16 +4,20 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title><%= htmlWebpackPlugin.options.title %></title>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">
+    <link rel="stylesheet" href="sample-volunteer-tasks.css">
+    <title>Cityzen Client Example</title>
   </head>
   <body>
     <noscript>
-      <strong>We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
+      <strong>
+        We're sorry but this test page doesn't work properly without
+        JavaScript enabled. Please enable it to continue.
+      </strong>
```

Comment:
We may want to look into doing some stuff with server side rendering so that we can support non-javascript clients

Reply:
SSR is usually straightforward to setup with Vue, but I'll have to think about how to handle the AJAX requests to the API in non-JS clients. 🤔